satellite/console/consoleweb/server.go

```diff
@@ -214,9 +219,7 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 	defer func() {
 		if err != nil {
 			server.log.Error("bucket usage report error", zap.Error(err))
```

Comment:
Because you have modified the line below, it's worth to remove this log call. `StatusNotFound` isn't an error in our system so we don't have to log it.